package.json

```diff
@@ -42,14 +42,14 @@
 		"debounce-fn": "^4.0.0",
 		"delay": "^4.4.0",
 		"delegate-it": "^2.0.0",
-		"dom-chef": "^4.2.0",
+		"dom-chef": "^4.2.2",
```

Comment:
If I remember correctly we have a todo with this one. (Something about in-line css)

Reply:
Solved? vadimdemedes/dom-chef#70

Reply:
https://github.com/sindresorhus/refined-github/blob/d8c1c195cfdff5031fda6dc688ba36828e9d7d49/source/features/remove-label-faster.tsx#L37